html/pfappserver/lib/pfappserver/Model/Enforcement.pm

```diff
@@ -27,6 +27,7 @@ my %types   = (
 #    option      => [ 'high-availability', 'dhcp-listener', 'monitor' ],
     webauth     => ['management', 'portal'],
     other       => ['dns-enforcement'],
+    radius      => ['management'],
```

Comment:
This is actually the 'required' types, since management contain radius it should not be necessary.